module-services/service-evtmgr/service-evtmgr/AppSettingsNotify.hpp

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
```

Comment:
Why is the file name AppSettingsNotify if it only notifies bluetooth audio settings?

Reply:
for some reason half of your comments are here twice :) https://github.com/mudita/MuditaOS/pull/3648#discussion_r827718886